src/Broker/Broker.h

```diff
@@ -0,0 +1,95 @@
+/**@file
+ * This file is part of the TASTE Runtime Common.
+ *
+ * @copyright 2021 N7 Space Sp. z o.o.
+ *
+ * TASTE Runtime Common was developed under a programme of,
+ * and funded by, the European Space Agency (the "ESA").
+ *
+ * Licensed under the ESA Public License (ESA-PL) Permissive,
+ * Version 2.3 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://essr.esa.int/license/list
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef BROKER_H
+#define BROKER_H
+
+/**
+ * @file    Broker.h
+ * @brief   Broker - message passing between nodes.
+ *
+ * The Broker is used by TASTE runtime to transport messages between nodes.
+ *
+ * The Runtime shall provide following functions:
+ * - void Broker_acquire_lock() - which locks the mutex designated to Broker
+ * - void Broker_release_lock() - which unlocks the mutex designated to Broker
+ *
+ * The Runtime shall provide following global variables:
+ * - driver_send_function bus_to_driver_send_function[SYSTEM_BUSES_NUMBER];
+ * - void* bus_to_driver_private_data[SYSTEM_BUSES_NUMBER];
+ * - deliver_function interface_to_deliver_function[INTERFACE_MAX_ID];
+ *
+ */
+
+#include <stddef.h>
+#include <stdint.h>
+
+#include <SpacePacket.h>
+
+#include <system_spec.h>
+#include <request_size.h>
+
+/// @brief  Buffer size to store space packet data.
+#define BROKER_BUFFER_SIZE                                                                                             \
+    (SPACE_PACKET_PRIMARY_HEADER_SIZE + GENERIC_LINUX_BUFFER_SIZE + SPACE_PACKET_ERROR_CONTROL_SIZE)
+
+/** @brief Initialize Broker
+ *
+ * Broker shall be initialized at the before starting threads of system.
```

Comment:
`initialized before starting system's threads`?

Reply:
done

Reply:
You forgot to remove `at the`

Reply:
true. fixed.